doc/user/layouts/partials/head.html

```diff
@@ -81,7 +81,7 @@
   data-modal-title="Materialize"
   data-modal-disclaimer="This is a custom LLM for Materialize with access to developer docs, guides and blog posts. This AI bot is experimental and might provide inaccurate or incomplete answers. Always consult the official Materialize documentation to verify any answers provided here. You can also switch to the Search mode to perform a search instead.
```

Comment:
Is there any way to squish the disclaimer down by a sentence or two? (Sorry I know this is complicated!)

Reply:
Let me try.

Reply:
So ... I think it's a no go.  Adding extra line breaks seems to have zero effect. 😞

Reply:
but removed the first sentence in the patch.